src/App.tsx

```diff
@@ -150,7 +150,7 @@ export default function App() {
           </Route>
           <Route path="/grid">
             <h2 className="section-title">Grid</h2>
-            <GridViewer></GridViewer>
+            <GridFull></GridFull>
```

Comment:
Tu peux aussi faire `<GridFull />` si tu sais que tu ne vas pas mettre de children

Reply:
Understood, to be replaced during pages refactoring.